lib/game.rb

```diff
@@ -0,0 +1,97 @@
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player1 = Player::Human.new("X"), player2 = Player::Human.new("O"), board = Board.new)
+    @player_1 = player1
+    @player_2 = player2
+    @board = board
+  end
+
+  WIN_COMBINATIONS = [
+    [0,1,2], #top row
+    [3,4,5], #middle row
+    [6,7,8], #bottom row
+    [0,3,6], #left column
+    [1,4,7], #middle column
+    [2,5,8], #right column
+    [0,4,8], #top left to bottom right
+    [6,4,2]  #bottom left to top right
+  ]
+
+  def current_player
+    (self.board.turn_count.even? ? self.player_1 : self.player_2)
+  end
+
+  def over?
+    (self.draw? || self.won? ? true : false)
```

Comment:
Avoid using the ternary operator to return `true` and `false`. Instead, rely on the implicit boolean return value of the expression. 

```
def over?
  self.draw? || self.won?
end
```